lib/kino/table.ex

```diff
@@ -3,14 +3,15 @@ defmodule Kino.Table do
 
   @type info :: %{
           name: String.t(),
-          features: list(:refetch | :pagination | :sorting)
+          features: list(:refetch | :pagination | :sorting | :filtering)
         }
 
   @type rows_spec :: %{
           offset: non_neg_integer(),
           limit: pos_integer(),
           order_by: nil | term(),
-          order: :asc | :desc
+          order: :asc | :desc,
+          filters: nil | list(%{String.t() => String.t()})
```

Comment:
`Kino.Table` has full control over the filter, so we should use a more specific type than `%{String.t() => String.t()}`. Something like `%{type : :equal | :less | ..., value: term(), column: term()}`.

Also, given it is a list now, we don't really need `nil`.

Sidenote: perhaps we should change order to `order: nil | %{direction: :asc | :desc, column: term()}` for consistency, wdyt? At this point it's fine to do it, since `Kino.Table` is not a public API yet :)

Reply:
Nice! Gonna do this in #237 coz I'll already be touching the front